Internal/domain/link/linkstorage.go

```diff
@@ -5,10 +5,10 @@ import (
 )
 
 type Link struct {
-	Link string
+	Link        string
 	ShortenLink string
-	Lifetime time.Duration
-	UserId *string
+	Lifetime    *time.Duration
+	AccountId   *string
```

Comment:
Do you really need a pointer here? Function signatures that use AccountId are inconsistent now.

Reply:
Pointer is needed because the user can be anonymous. By default, we assume that the pointer is nil.

Reply:
Ok, I need to think it over..